plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -36,7 +36,8 @@ type TimeFunc func() time.Time
 // HTTPListenerV2 is an input plugin that collects external metrics sent via HTTP
 type HTTPListenerV2 struct {
 	ServiceAddress string            `toml:"service_address"`
-	Path           string            `toml:"path"`
+	Paths          []string          `toml:"paths"`
+	PathTag        string            `toml:"path_tag"`
```

Comment:
Please change this to
```suggestion
	PathTag        bool            `toml:"path_tag"`
```